### PR TITLE
Stackdriver logging samples (JUL, Logback, Client library)

### PR DESCRIPTION
**Pull request by jabubake**
- Also updated List Logs snippets to use `google-cloud-java`

### CODE REVIEW

File: logging/cloud-client/src/main/java/com/example/logging/QuickstartSample.java [hunk 1]
```diff
@@ -40,11 +47,12 @@ public static void main(String... args) throws Exception {
     String text = "Hello, world!";
 
     LogEntry entry = LogEntry.newBuilder(StringPayload.of(text))
+        .setSeverity(Severity.ERROR)
         .setLogName(logName)
         .setResource(MonitoredResource.newBuilder("global").build())
         .build();
 
-    // Writes the log entry
+    // Writes the log entry asynchronously
```
Comment (lesv) [inline logging/cloud-client/src/main/java/com/example/logging/QuickstartSample.java]: Out of curiosity, do we do anything to improve the odds that logs get written even if the app crashes? Can we? Big try / catch in main?

File: logging/jul/src/main/java/com/example/logging/jul/Quickstart.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+  Copyright 2017, Google, Inc.
```
Comment (lesv) [inline logging/jul/src/main/java/com/example/logging/jul/Quickstart.java]: Don't think the "," is nesc.

File: logging/logback/src/main/java/com/example/logging/logback/Quickstart.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/*
+  Copyright 2017, Google, Inc.
```
Comment (lesv) [inline logging/logback/src/main/java/com/example/logging/logback/Quickstart.java]: comma

File: logging/cloud-client/src/main/java/com/example/logging/QuickstartSample.java (hunk 1 above)
Comment (jabubake) [reply logging/cloud-client/src/main/java/com/example/logging/QuickstartSample.java]: we do offer a logging.flush() but would n't help with the try / catch in main. 
I get your point though, so will look into it separately as an issue on `google-cloud-java`.

File: logging/jul/src/main/java/com/example/logging/jul/Quickstart.java (hunk 2 above)
Comment (jabubake) [reply logging/jul/src/main/java/com/example/logging/jul/Quickstart.java]: Done. Fixed missing licenses too.

File: logging/logback/src/main/java/com/example/logging/logback/Quickstart.java (hunk 3 above)
Comment (jabubake) [reply logging/logback/src/main/java/com/example/logging/logback/Quickstart.java]: Done.

**Pull request merged**